### ISSUE TITLE
Cannot connect to sever when the player is an operator

### ISSUE THREAD

**Issue by SilianZ** [NGoedix/VideoPlayer#42]
Mod version: 2.6
Watermedia version: 2.0.27
```
Unrecognized argument type com.github.NGoedix.videoplayer.commands.arguments.SymbolStringArgumentType@13a531f (class com.github.NGoedix.videoplayer.commands.arguments.SymbolStringArgumentType)
[Server]        at net.minecraft.class_2316.method_41983(class_2316.java:147) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2316.method_41985(class_2316.java:153) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2641$class_7232.<init>(class_2641.java:162) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2641.method_11401(class_2641.java:235) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2641.method_30945(class_2641.java:107) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2641.<init>(class_2641.java:52) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2170.method_9241(class_2170.java:392) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_3324.method_14596(class_3324.java:650) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_3324.method_14576(class_3324.java:551) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_3324.method_14556(class_3324.java:531) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_3244.method_12068(class_3244.java:1590) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2799.method_12120(class_2799.java:24) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2799.method_11054(class_2799.java:6) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_2600.method_11072(class_2600.java:24) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_3738.run(class_3738.java:18) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_4093.mixinextras$bridge$method_18859$5(class_4093.java) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_4093.wrapOperation$bni000$carpet-tis-addition$yeetUpdateSuppressionCrash_implForThreadExecutorTaskExecuting(class_4093.java:570) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:797) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:165) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:779) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:773) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.class_1255.method_18857(class_1255.java:139) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:758) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:687) ~[server-intermediary.jar:?]
[Server]        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) ~[server-intermediary.jar:?]
[Server]        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by IamKsandr707** [comment]
Same

**Comment by ALxR1337** [comment]
Same. Its working in singleplayer, but not on server. Fabric 1.20.4
Even if I join the server and try to op me again, it says an unexpected error occurred trying to execute that command.

**Comment by NGoedix** [comment]
Currently working on the 3.0 with a fix

**Comment by NGoedix** [comment]
Fixed on 3.0.1 (available on modrinth)
